src/cat.ts

```diff
@@ -100,6 +115,38 @@ export class Cat {
     return this._zetas;
   }
 
+  public get prior() {
+    return this._prior;
+  }
+
+  private static validatePrior(priorDist: string, priorPar: number[], minTheta: number, maxTheta: number) {
+    if (priorDist !== 'norm' && priorDist !== 'unif') {
+      throw new Error('The prior distribution you provided is not supported');
+    }
+    if (priorDist === 'norm') {
+      if (priorPar.length !== 2) {
+        throw new Error('The prior distribution parameters you provided are not valid');
+      }
```

Comment:
This check is the same for both `'norm'` and `'unif'` so it could be moved outside of the `if (priorDist === 'norm')` clause.

Also, I'd suggest a more explicit error message like
```
throw new Error('The prior distribution parameters should be an array of two numbers. Received ${priorPar}.');
```